context_prop/src/main/java/io/opentelemetry/context/propagation/HttpTextFormat.java

```diff
@@ -69,15 +69,17 @@
   List<String> fields();
 
   /**
-   * Injects the value downstream. For example, as http headers.
+   * Injects the value downstream. For example, as http headers. The carrier may be null to
+   * facilitate calling this method with a java lambda for the {@link Setter}, in which case that
```

Comment:
```suggestion
   * facilitate calling this method with a lambda for the {@link Setter}, in which case that
```

Java is redundant in a Javadoc, IMHO.

Reply:
fair. :)